go/kubebuilder/scaffold_go_operator.sh

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env bash
+
+NO_COLOR=${NO_COLOR:-""}
+if [ -z "$NO_COLOR" ]; then
+  header_color=$'\e[1;33m'
+  reset_color=$'\e[0m'
+else
+  header_color=''
+  reset_color=''
+fi
+
+OLD_RECONCILER_IDENTIFIER='kubebuilder'
```

Comment:
Could we have it required replace in a file with the name of the replace?
E.g reconcile_replaces and type_replaces

Reply:
Did you mean renaming the variables to include the filename too ?